README.md

```diff
@@ -8,7 +8,41 @@ A library to **sub**mit e**xt**rinsics to a [substrate](https://github.com/parit
 
 ## Usage
 
-See [examples](./examples).
+### Downloading metadata from a Substrate node
+
+Use the `subxt-cli` CLI tool to download the metadata for your target runtime from a Substrate node.
+
+```bash
+cargo run -p subxt-cli -- metadata -f bytes > metadata.scale
+```
+
+This defaults to querying the metadata of a locally running node on the default `http://localhost:9933/`. If querying
+a different node then the `metadata` command accepts a `--url` argument.
+
+### Generating the runtime API from the downloaded metadata
+
+Simply declare a module and decorate it with the `subxt` attribute which points at the downloaded metadata for the 
+target runtime:
+
+```rust
+#[subxt::subxt(runtime_metadata_path = "metadata.scale")]
+pub mod node_runtime { }
+```
+
+**Important:** `runtime_metadata_path` resolves to a path relative to the directory where your project's `Cargo.toml` 
```

Comment:
dq: relative the *workspace* Cargo.toml or the current crate's Cargo.toml?

Reply:
Ah whatever `std::env::var("CARGO_MANIFEST_DIR")` resolves to, which is would be the workspace I believe.

Reply:
I was curious so I had a play; it looks like, however you try compiling a crate in a workspace, CARGO_MANIFEST_DIR is always relative to the crate itself and not the workspace.

Which makes sense I think :)

Reply:
Ah yes the clue is in the name!